.markdownlint/style.rb

```diff
@@ -18,7 +18,6 @@
 rule 'MD021'
 rule 'MD022'
 rule 'MD023'
-rule 'MD024'
```

Comment:
Don't remove it, we can use:  https://github.com/markdownlint/markdownlint/pull/200/files

```
rule 'MD024', :allow_different_nesting => true
```